lib/DocumentsFieldArray/FileUploaderField.js

```diff
@@ -45,6 +47,7 @@ class FileUploaderField extends React.Component {
       isDropZoneActive: false,
     });
 
+    this.setState({ uploadInProgress: true });
```

Comment:
You can just loop this into the `setState` above, no?

Reply:
Oh yeah, I missed that block of code somehow. Will add it in there.